src/Rating/Rating.tsx

```diff
@@ -69,9 +69,10 @@ export const Rating: FC<RatingProps> = ({
   const onChange = readOnly ? noop : onChangeRating;
 
   const wrapperClasses = classNames(wrapperClassName, {
-    'rating-read-only': readOnly,
-    'rating-label': legend
+    'rating rating-read-only': readOnly,
```

Comment:
+1 for this. Without the suggested change, the `rating` class is added only if at least one of `legend` and `readOnly` is set. If neither of the two is set, the component style is completely broken because the class is missing.

Reply:
fixed in #937